packages/core/parcel-bundler/src/Bundler.js

```diff
@@ -103,6 +103,12 @@ class Bundler extends EventEmitter {
           : watch;
     const scopeHoist =
       options.scopeHoist !== undefined ? options.scopeHoist : false;
+    const autoInstall =
+      typeof options.autoinstall === 'boolean'
+        ? options.autoinstall
+        : typeof process.env.PARCEL_AUTOINSTALL === 'boolean'
```

Comment:
does that actually get converted to a boolean if you provide this environment variable from a terminal?